src/main/java/uniresolver/driver/did/factom/DIDFactomDriver.java

```diff
@@ -70,37 +93,40 @@ public ResolveResult resolve(String identifier) throws ResolutionException {
 
 
         try {
-            IdentityResponse identityResponse = getClient(networkId).getIdentityResponse(targetIdentifier, EntryValidation.IGNORE_ERROR, Optional.empty(), Optional.empty());
-            DIDDocument didDocument = getClient(networkId).factory().toDid(identifier, identityResponse);
-            ResolveResult resolveResult = ResolveResult.build(didDocument);
-            resolveResult.setMethodMetadata(createMethodMetadata(identifier, networkId, identityResponse, didDocument, start));
-            resolveResult.setResolverMetadata(createResolverMetadata(identifier, identityResponse, didDocument, start));
-            return resolveResult;
-        } catch (RuleException e) {
+            IdentityClient client = getClient(networkId);
+            List<FactomIdentityEntry<?>> allEntries = client.lowLevelClient()
+                    .getAllEntriesByIdentifier(identifier, EntryValidation.THROW_ERROR, Optional.empty(), Optional.empty());
```

Comment:
Do we really want to throw errors on an invalid entry? The previous one was ignore errors, which makes more sense to me (skipping invalid entries altogether on resolution, according to the Factom DID spec)

Reply:
updated to ignore.